.github/FUNDING.yml

```diff
@@ -0,0 +1,12 @@
+# These are supported funding model platforms
+
+github: # Replace with up to 4 GitHub Sponsors-enabled usernames e.g., [user1, user2]
+patreon: # Replace with a single Patreon username
+open_collective: # Replace with a single Open Collective username
+ko_fi: # Replace with a single Ko-fi username
+tidelift: # Replace with a single Tidelift platform-name/package-name e.g., npm/babel
+community_bridge: # Replace with a single Community Bridge project-name e.g., cloud-foundry
+liberapay: # Replace with a single Liberapay username
+issuehunt: bertfrees # Replace with a single IssueHunt username
```

Comment:
Why this one? Have you researched them? Are they legit and morally acceptable? I guess so otherwise presumably github would not support them.

Reply:
This was the only one that works with bounties.

Reply:
OK. Why do you think bounties are important? I think we should just generally test sponsoring/funding.

Reply:
I have also applied for the Github sponsoring. But I think what is nice about bounties is that we can also reward other contributors, and this way we also possibly attract more developers.

Reply:
OK, makes sense